src/sentry/api/endpoints/organization_event_details.py

```diff
@@ -59,7 +67,8 @@ def get(self, request, organization, project_slug, event_id):
         data = serialize(event)
         data['nextEventID'] = next_event[1] if next_event else None
         data['previousEventID'] = prev_event[1] if prev_event else None
-
+        data['oldestEventID'] = self.oldest_event_id(snuba_args, event)
```

Comment:
Will this be replacing the oldest/latest endpoints everywhere then? The thing I liked about supporting the "oldest" and "latest" aliases was that we remove the need to do these 2 extra queries.

Reply:
I was thinking of removing the endpoints. This does add two extra snuba queries but simplifies the client code and reduces the number of endpoints and tests we have to maintain in the long term.